go.mod

```diff
@@ -12,14 +12,15 @@ require (
 	github.com/apache/thrift v0.0.0-20161221203622-b2a4d4ae21c7
 	github.com/aws/aws-sdk-go v1.15.68 // indirect
 	github.com/bmizerany/perks v0.0.0-20141205001514-d9a9656a3a4b // indirect
-	github.com/census-instrumentation/opencensus-proto v0.1.0
+	github.com/census-instrumentation/opencensus-proto v0.0.3-0.20181214143942-ba49f56771b8
```

Comment:
Please don't forget to run the go get -u for the proto repository, as this proto version should be `v0.1.x`

Reply:
It turns out that the `GO111MODULE=on go get -u -v github.com/census-instrumentation/opencensus-proto@master` still didn't make it `v0.1.0-...`, but I was able to manually update it to `v0.1.0-0.20181214143942-ba49f56771b8` and it worked.

Reply:
Note that manual edits of this sort will no longer work in Go 1.13.
(And users may have to apply a `replace` directive to correct manually-edited pseudo-versions in the wild.)

Going forward, a more robust fix for this sort of problem is to add an explicit pre-release tag upstream (such as `v0.1.0-dev`) so that the `go` command can deduce the correct relationship between the release branch and the master branch.